src/MessageFormatter.php

```diff
@@ -4,6 +4,10 @@
 
 namespace Yiisoft\Yii\DataView;
 
+use function is_array;
+use function is_object;
+use function is_resource;
+
 class MessageFormatter
```

Comment:
I think current translator has that already: https://github.com/yiisoft/translator#message-formatters

Reply:
done.